index.js

```diff
@@ -1,3 +1,4 @@
+require('babel-plugin-check-es2015-constants');
```

Comment:
I know what the plugin does. I'm saying that requiring it returns the plugin. It must be present in the export of this module to be used. Just requiring it as a statement does nothing. Also, even if it did something, you would not want to run it if your environment had proper support for const semantics.